MULTIPLE_STRIP_TESTING_WORKS_MIDI/Color_Edit.ino

```diff
@@ -1,76 +1,150 @@
 void coloredit () {
 
-  static byte pos = 0;
+  //static byte pos = 0;
   encoder.tick();
   RotaryEncoder::Direction dir = encoder.getDirection();
 
-  byte newPos = encoder.getPosition();
-  if (pos != newPos) {
+  byte newcolorPos = encoder.getPosition();
+  if (colorpos != newcolorPos) {
     //Serial.print("pos:");
     //Serial.print(newPos);
     //Serial.print(" dir:");
     //Serial.println((int)(encoder.getDirection()));
-    pos = newPos;
+    colorpos = newcolorPos;
     //delay (20);
   }
-      fill_solid(tom1strip, TOM1LEDS, CHSV(0, 0, 0));
-      fill_solid(tom2strip, TOM2LEDS, CHSV(0, 0, 0));
-      fill_solid(tom3strip, TOM3LEDS, CHSV(0, 0, 0));
-      fill_solid(snarestrip, SNARELEDS, CHSV(0, 0, 0));
-      fill_solid(bassstrip, BASSLEDS, CHSV(0, 0, 0));
-      fill_solid(hihatstrip, HIHATLEDS, CHSV(0, 0, 0));
-      fill_solid(crashstrip, CRASHLEDS, CHSV(0, 0, 0));
-      fill_solid(ridestrip, RIDELEDS, CHSV(0, 0, 0));
-      controllers[0]->showLeds();
-      controllers[1]->showLeds();
-      controllers[2]->showLeds();
-      controllers[3]->showLeds();
-      controllers[4]->showLeds();
-      controllers[5]->showLeds();
-      controllers[6]->showLeds();
-      controllers[7]->showLeds();
-
-
   switch (padHit) {
     case 1:
+<<<<<<< Updated upstream
+<<<<<<< Updated upstream

```

Comment:
Oops - Looks like you have hit a conflict. These will break your code. They are annotations from GIT and you have to resolve these. I will have a go at cleaning this up.

Reply:
Not sure what I did there. Still figuring out the Git hub side of things.

Reply:
Yes, the conflict resolution is a pain - especially with the editor does not handle it automatically.